pycmr/queries.py

```diff
@@ -240,6 +241,60 @@ def day_night_flag(self, day_night_flag):
         self.params['day_night_flag'] = day_night_flag
         return self
 
+    def cloud_cover(self, min_cover=0, max_cover=100):
+        """
+        Set the cloud cover value for the query
+        """
+
+        if not min_cover and not max_cover:
+            raise ValueError("Please provide atleast min_cover, max_cover or both")
+
+        if min_cover and max_cover:
+            try:
+                minimum = float(min_cover)
+                maxiumum = float(max_cover)
+
+                if minimum > maxiumum:
+                    raise ValueError("Please ensure min cloud cover is lower than max cloud cover")
```

Comment:
For consistency with the other messages, probably should add the underscores to the parameter names.